hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerDoubleBufferWithOMResponse.java

```diff
@@ -345,21 +345,23 @@ public void testDoubleBuffer(int iterations, int bucketCount)
       }
 
       // We are doing +1 for volume transaction.
-      GenericTestUtils.waitFor(() ->
-              doubleBuffer.getFlushedTransactionCount() ==
-                  (bucketCount + 1) * iterations, 100,
-          120000);
+      long expectedTransactions = (bucketCount + 1) * iterations;
+      GenericTestUtils.waitFor(() -> lastAppliedIndex == expectedTransactions,
+          100, 120000);
 
-      Assert.assertTrue(omMetadataManager.countRowsInTable(
-          omMetadataManager.getVolumeTable()) == iterations);
+      Assert.assertEquals(expectedTransactions,
+          doubleBuffer.getFlushedTransactionCount()
+      );
 
-      Assert.assertTrue(omMetadataManager.countRowsInTable(
-          omMetadataManager.getBucketTable()) == (bucketCount) * iterations);
+      Assert.assertEquals(iterations,
+          omMetadataManager.countRowsInTable(omMetadataManager.getVolumeTable())
+      );
 
-      Assert.assertTrue(doubleBuffer.getFlushIterations() > 0);
+      Assert.assertEquals(bucketCount * iterations,
+          omMetadataManager.countRowsInTable(omMetadataManager.getBucketTable())
+      );
 
-      // Check lastAppliedIndex is updated correctly or not.
-      Assert.assertEquals((bucketCount + 1) * iterations, lastAppliedIndex);
```

Comment:
Why this check lastAppliedIndex is removed?

Reply:
The waitFor() moved above should have guarantee this condition already.

Reply:
Yes, you are right, I have missed it. Thanks for the pointer.